products.css

```diff
@@ -0,0 +1,137 @@
+body {
+    background-color: white;
+    color: black;
+    font-family: "comic sans ms";
+
+}
+
+.container {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+}
+
+.nav-container {position:fixed;
+    left : 1vw;
+   z-index: 1;
+    display: flex;
+    flex-grow: 1;
+    background-color: rgb(188, 251, 62);
+    align-items: center;
+    justify-content: space-between;
+    width: 97vw;
+
+}
+
+.nav-footer {
+    align-items: center;
+    width: 20%;
+}
+
+.btn-nav {
+    
+    margin: 0px;
+    background-color: lightgray;
```

Comment:
there is extra space and you can use variables here too.
Also you can specify margin :0 at the root so you won't have to mention it everywhere